src/main/telemetry/jetiexbus.c

```diff
@@ -173,8 +204,33 @@ void initJetiExBusTelemetry(void)
     jetiExTelemetryFrame[EXTEL_HEADER_LSN_LB] = 0x00;            // increment by telemetry count (%16) > only 15 values per device possible
     jetiExTelemetryFrame[EXTEL_HEADER_LSN_HB] = 0x00;
     jetiExTelemetryFrame[EXTEL_HEADER_RES] = 0x00;               // reserved, by default 0x00
+
+
+    // Check which sensors are available
+
+    if (batteryConfig()->voltageMeterSource != VOLTAGE_METER_NONE) jetiExSensors[EX_VOLTAGE].parameter = EX_TYPE_SENSOR_ENABLED;
+    if (batteryConfig()->currentMeterSource != CURRENT_METER_NONE) jetiExSensors[EX_CURRENT].parameter = EX_TYPE_SENSOR_ENABLED;
+    if ((batteryConfig()->voltageMeterSource != VOLTAGE_METER_NONE) && (batteryConfig()->currentMeterSource != CURRENT_METER_NONE)) {
+        jetiExSensors[EX_POWER].parameter = EX_TYPE_SENSOR_ENABLED;
+    }       jetiExSensors[EX_CAPACITY].parameter = EX_TYPE_SENSOR_ENABLED;
```

Comment:
Here it seems something is wrong. Maybe the open `{` and one `enter` ?

Reply:
You are right. I should never use diffrent editors...